HW2/P2/P2.py

```diff
@@ -38,14 +38,15 @@
 
```

Comment:
Fancy plot! That would be great if the label of legends could be more detailed (like "Medium-grained locking in uncorrelated transfers").